Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeDrawingAreaProxyIOS.mm

```diff
@@ -207,14 +207,39 @@ - (void)didChangeNominalFramesPerSecond
     [displayLinkHandler() setPreferredFramesPerSecond:preferredFramesPerSecond];
 }
 
+void RemoteLayerTreeDrawingAreaProxyIOS::didRefreshDisplay()
+{
+    if (m_needsDisplayRefreshCallbacksForDrawing)
+        RemoteLayerTreeDrawingAreaProxy::didRefreshDisplay();
+
+    if (m_needsDisplayRefreshCallbacksForAnimation)
+        m_webPageProxy->scrollingCoordinatorProxy()->displayDidRefresh(0);
```

Comment:
The `0` is mysterious.

We do actually synthesize fake displayIDs for iOS (see `generateDisplayIDFromPageID()`). You should use the correct value.

Reply:
Fixed with latest iteration of this PR.